### ISSUE TITLE
Add ability to remove a plugin from within Protege

### ISSUE THREAD

**Issue by rsgoncalves** [protegeproject/protege#648]

**Comment by ThomasThelen** [comment]
I ran across this issue while searching for a way to disable a particular plugin.

In terms of user interface, I think a fourth column next to _Qualifier_, named _Enabled_ would be sensible placement. When selected, the plugin is enabled; when unselected, the plugin is disabled. Is anyone willing to provide insight on what the backing model is where the state of each plugin can be stored?? 

<img width="931" alt="Screen Shot 2021-11-06 at 4 35 49 PM" src="https://user-images.githubusercontent.com/9289652/140626946-931465f5-e33d-405b-8902-55d8859b63e4.png">


Here's my understanding of how this would function from a user's workflow

Not saving:

1. User enables/disables plugins
2. User clicks 'Cancel'
3. Plugin state is not changed
4. User closes & opens Protege
5. User sees that their plugin changes haven't been saved

Saving:

1. User enables/disables plugins
2. User clicks 'OK'
3. Plugin state is saved globally
4. User closes & opens Protege
5. User sees their plugin changes have persisted 


Things to do:

1. Add the new column; make sure it correctly reflects the state of the backing model
1. Correct checkbox state when user selects/deselects & clicks OK/Cancel
1. Modify the backing model to support the state of each plugin
1. Persist the new changes to the model on disk
1. Load the states on disk (while also loading previous formats without the state)

Questions:

1. What happens when the user clicks 'reset defaults'? Disable all the plugins?

**Comment by rpgoldman** [comment]
In the short term, just telling us where to find the plugin files so that we can remove them would be a huge help. This could be done just by adding a sentence to the Plugins preference pane  for Protege.

**Comment by mdebellis** [comment]
I would like to do this too. Is there any way to uninstall a plugin right now? I think you can reset the plugins to default but then I have to reload all the plugins I want.

**Comment by gouttegd** [comment]
No, I don’t think there is a way to “uninstall” a plugin directly from within Protégé for now. I agree that’d be nice.

> just telling us where to find the plugin files so that we can remove them would be a huge help.

If your user account has write access to Protégé’s installation directory, plugins are installed in the `plugins` subdirectory (for example, on macOS, and assuming Protégé is installed normally in `/Applications`, plugins would be in `/Applications/Protégé.app/Contents/plugins`).

If you do not have write access to Protégé’s installation directory, plugins are installed in your home directory, under `~/.Protege/plugins`.

**Comment by mdebellis** [comment]
Thanks. I'm on Windows and I found my plugin directory and there are a bunch of jar files. So if I want to remove a plugin, I assume all I need to do is remove the jar file with the name of that plugin and when I restart Protege the plugin will no longer be there. Is that correct? (Just want to be sure I don't mess something up)

**Comment by gouttegd** [comment]
Yes, that’s correct.
